Makefile

```diff
@@ -15,6 +15,9 @@
 # If you update this file, please follow
 # https://suva.sh/posts/well-documented-makefiles
 
+# Ensure Make is run with bash shell as some syntax below is bash-specific
+SHELL :=/bin/bash
```

Comment:
```suggestion
SHELL:=/usr/bin/env bash
```

Reply:
Use `/usr/bin/env bash`, which is more compatible across systems.
ref: https://github.com/kubernetes/release/blob/ffbc45e750bdd123aefacf70e6727a21701c76d2/Makefile#L19

Reply:
Thanks @justaugustus, addressed in new commit